README.md

```diff
@@ -3,32 +3,119 @@
 [![npm](https://badgen.net/npm/v/is-scrolled)](https://www.npmjs.com/package/is-scrolled)
 [![npm](https://badgen.net/npm/dt/is-scrolled)](https://www.npmjs.com/package/is-scrolled)
 [![npm](https://badgen.net/npm/dm/is-scrolled)](https://www.npmjs.com/package/is-scrolled)
+[![Snyk Vulnerabilities for GitHub Repo](https://img.shields.io/snyk/vulnerabilities/github/bibixx/is-scrolled)](https://snyk.io/test/github/bibixx/is-scrolled)
 ![CircleCI branch](https://badgen.net/circleci/github/bibixx/is-scrolled)
 [![david-dm.org](https://badgen.net/david/dep/bibixx/is-scrolled)](https://david-dm.org/bibixx/is-scrolled)
 [![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg)](https://github.com/semantic-release/semantic-release)
 [![Commitizen friendly](https://badgen.net/badge/commitizen/friendly/green)](http://commitizen.github.io/cz-cli/)
 
-> TBA
+> React component & hook to easily check whether whether scroll area has reached its limit
 
 ## Installation
 
 `npm install -S is-scrolled`
 
 ## Examples
```

Comment:
Love it that you have a playground and simple examples.
I noticed while using the playground that my framerate dropped. I suggest running some benchmarks and maybe think of throttling 'onscroll' handler.